Upstream: Yes (http://cups.org/str.php?L3347+P0+S-2+C0+I0+E0+M20+Q)
Reason: Do not try to access xinetd if it's not available
Source: Daniel Mierswa <impulze@impulze.org>

diff -Naur cups-1.4.1.bak/config-scripts/cups-defaults.m4 cups-1.4.1/config-scripts/cups-defaults.m4
--- cups-1.4.1.bak/config-scripts/cups-defaults.m4	2009-02-10 18:05:35.000000000 +0100
+++ cups-1.4.1/config-scripts/cups-defaults.m4	2009-09-25 14:49:20.000000000 +0200
@@ -325,8 +325,8 @@
 				CUPS_DEFAULT_LPD_CONFIG_FILE="launchd:///System/Library/LaunchDaemons/org.cups.cups-lpd.plist"
 				;;
 			*)
-				if test -d /etc/xinetd.d; then
-					CUPS_DEFAULT_LPD_CONFIG_FILE="xinetd:///etc/xinetd.d/cups-lpd"
+				if test "x$XINETD" != x; then
+					CUPS_DEFAULT_LPD_CONFIG_FILE="xinetd://$XINETD/cups-lpd"
 				else
 					CUPS_DEFAULT_LPD_CONFIG_FILE=""
 				fi
